src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -125,13 +125,16 @@ const TagsRenderer = (data: DashboardInterface) => {
       })}
       {data?.description &&
         data.type === DashboardType.ACCOUNT &&
-        data?.description.split(',').map((tag: string, index: number) => {
-          return (
-            <section className={css.customTag} key={tag + index}>
-              {tag}
-            </section>
-          )
-        })}
+        data?.description
+          .split(',')
+          .filter((tag: string) => tag !== '')
+          .map((tag: string, index: number) => {
+            return (
+              <section className={css.customTag} key={tag + index}>
+                {tag}
+              </section>
+            )
+          })}
```

Comment:
Absolutely nothing wrong with this `map`, but you can drop a few characters for things like this with an implicit return.
```suggestion
          .map((tag: string, index: number) => (
            <section className={css.customTag} key={tag + index}>
              {tag}
            </section>
          ))}
```